src/streaming/controllers/PlaybackController.js

```diff
@@ -270,7 +271,7 @@ MediaPlayer.dependencies.PlaybackController = function () {
 
             // do nothing if common earliest time has not changed or if the firts segment has not been appended or if current
             // time exceeds the common earliest time
-            if ((currentEarliestTime === commonEarliestTime[id]) || !firstAppended.ready || (time > commonEarliestTime[id])) return;
+            if ((currentEarliestTime === commonEarliestTime[id] && (time === currentEarliestTime)) || !firstAppended.ready || (time > commonEarliestTime[id])) return;
```

Comment:
@KarishmaBagga, sure. It is possible that both audio and video have the same earliest time, which > time (time = current time). For example, time = 0, earliestAudio = earliestVideo = 0.2. Without the check that has been added the player would not seek to 0.2 while it should.